spec/Records.tex

```diff
@@ -272,28 +187,25 @@ \subsection{Record Initialization}
 \index{initialization!record}
 
 A variable of a record type declared without an initialization expression
-is initialized through a call to the record's default initializer, passing no arguments.
-The default initializer for a record is defined in the same way as the default
-initializer for a class (\rsec{Default_Initialization}).
-
-If the new record type is derived from other record types, the
-default initializer for each base record will be called in lexical order before default
-initializer for the record itself.
+is initialized through a call to the record's default initializer,
+passing no arguments.  The default initializer for a record is defined in
+the same way as the default initializer for a class
+(\rsec{Default_Initialization}).
 
 To construct a record as an expression,
 i.e. without binding it to a variable, the \chpl{new} operator is
 required.  In this case, storage is allocated and reclaimed as for a record
 variable declaration (\rsec{Record_Storage}), except that the temporary record
 goes out of scope at the end of the enclosing expression.
```

Comment:
Above in "at the end of the enclosing expression", let's change "expression" to "statement".